client/src/pages/Login.jsx

```
@@ -25,21 +31,45 @@ export default function Login() {
-        onClick={() => (window.location.href = NAVER_AUTH_URL)}
+        onClick={() => {
+          localStorage.setItem('lastLoginProvider', 'naver');
+          window.location.href = NAVER_AUTH_URL;
+        }}
       >
         <img src={naver} alt="naverLogin" className={styles.loginButtonImage} />
+        {lastProvider === 'naver' && (
+          <div className={styles.highlighted} onClick={(e) => e.stopPropagation()}>
+            최근 로그인
+          </div>
+        )}
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)